sdcm/kcl_thread.py

```diff
@@ -38,26 +37,16 @@ def run(self):
         return _self
 
     def build_stress_cmd(self):
-        if self.params.get('alternator_use_dns_routing'):
-            target_address = 'alternator'
+        if getattr(self.node_list[0], 'parent_cluster'):
```

Comment:
`getattr(x, 'y')` is equivalent to `x.y`. You need to provide default value if you're expecting that there can be no such property:

```
getattr(...)
    getattr(object, name[, default]) -> value
    
    Get a named attribute from an object; getattr(x, 'y') is equivalent to x.y.
    When a default argument is given, it is returned when the attribute doesn't
    exist; without it, an exception is raised in that case.
```

Reply:
yes I guess the correct thing is `hasattr`, but it's weird... cause it was working.
i.e. it's only for some integrations test, that doesn't mock all the grid of classes of a cluster. (maybe it's not needed anymore)
I'll check

Reply:
fixed it